plinth/modules/users/forms.py

```diff
@@ -149,14 +153,21 @@ class Meta:
 
     def __init__(self, request, username, *args, **kwargs):
         """Initialize the form with extra request argument."""
-        for group, group_name in GROUP_CHOICES:
+        group_choices = dict(get_group_choices())
+        for group in group_choices:
             Group.objects.get_or_create(name=group)
 
         self.request = request
         self.username = username
-
         super(UserUpdateForm, self).__init__(*args, **kwargs)
 
+        choices = [  # Replace group names with descriptions
+            (c[0], group_choices[c[1]])
+            for c in sorted(self.fields['groups'].choices, key=lambda x: x[1])]
```

Comment:
I get an error here when testing:
```
File "/vagrant/plinth/modules/users/forms.py" in <listcomp>
  166.             for c in sorted(self.fields['groups'].choices, key=lambda x: x[1])]

Exception Type: KeyError at /sys/users/tester/edit/
Exception Value: 'wiki'
```
To get past this error, I had to install ikiwiki, then restart plinth.